### ISSUE TITLE
SGit Crashed while updating references

### ISSUE THREAD

**Issue by xxoscarxx28** [sheimi/SGit#135]
Everytime I try to clone a repository... SGit crashes when it says that it updates references... And never finishes if you tried to continue.

**Comment by d10** [comment]
Can I get your Android version? And did you enable ART?

**Comment by xxoscarxx28** [comment]
Im using LG G2 with CloudyG3 2.1.0 which is an Android 4.4.2, I switched to
ART Runtime... So SGit is not compatible with ART Runtime?

**Comment by d10** [comment]
Yep, I think there are some issues with JGit, which we use for Git operations when using ART on Android 4.4. This is fixed in Android 5’s ART implementation. See #103.

**Comment by xxoscarxx28** [comment]
Then why not fix the issues with JGit for ART implementations of Android 4.4.2 ???

**Comment by d10** [comment]
About that:

1. We all have limited time that we can spend with our OSS libraries.
2. JGit is a library that we use, and I personally am not too sure about
how it's implemented.
3. I don't have an issue with ART, so it's hard to debug.

TL;DR: I have no time to do it. Feel free to send PRs if you are able to
solve it.

**Comment by Dentedbowl** [comment]
I want to unsubscribe from any and all email and or text messages from this website thanks

-----Original Message-----
